test/ignore.spec.js

```diff
@@ -0,0 +1,48 @@
+import Alpine from 'alpinejs'
+import { wait } from '@testing-library/dom'
+
+global.MutationObserver = class {
+    observe() {}
+}
+
+test('x-ignore ignores initialization of any new components within the node', async () => {
+    document.body.innerHTML = `
+    <div x-ignore="ignore" data="{ignore:true}">
+        <div x-data="{foo:'foo'}">
```

Comment:
Will it still work if we do the following?

```suggestion
    <div x-ignore="ignore" x-data="{foo:'foo'}">
        <div>
```

Reply:
Yes, i actually tried bunch of ways but i think real world usages are more important. I have included similar one to this in the last update.